cli/rt/run.rs

```diff
@@ -592,6 +592,11 @@ impl ModuleLoader for EmbeddedModuleLoader {
     Some(Cow::Owned(data.data.to_vec()))
   }
 
+  fn source_map_source_exists(&self, source_url: &str) -> Option<bool> {
+    let url = Url::parse(source_url).ok()?;
+    Some(self.shared.modules.read(&url).ok().flatten().is_some())
```

Comment:
I suppose that this is enough for EmbeddedModuleLoader. No `fs exists` here, right?
cc @dsherret and @bartlomieju